WordPress/Classes/ViewRelated/NUX/SiteCreationSitePreviewViewController.swift

```diff
@@ -0,0 +1,94 @@
+import UIKit
+import WebKit
+
+class SiteCreationSitePreviewViewController: UIViewController {
+
+    // MARK: - Properties
+
+    var siteUrl: String?
+
+    @IBOutlet weak var siteView: UIView!
+    @IBOutlet weak var congratulationsView: UIView!
+    @IBOutlet weak var congratulationsLabel: UILabel!
+    @IBOutlet weak var siteReadyLabel: UILabel!
+
+    /// Due to build error when adding WKWebView in a storyboard, must create WKWebView
```

Comment:
Ah, interesting.

Maybe we should just create _everything_ programatically? *ducks*

*runs away*